### ISSUE TITLE
Prime Number Test.cpp

### ISSUE THREAD

**Issue by gksbrandon** [chiriacandrei25/The-Bible-of-Algorithms#1]
The for loop should be i <= sqrt(n), if not, you'd get True for numbers like 9
`#include <iostream>
#include <cmath> /// We need the "cmath" library for the sqrt function

using namespace std;

bool isPrime(int n) {
    if(n <= 1)
        return false;
    for(int i = 2; i < sqrt(n); i++) {
        if(n % i == 0)
            return false;
    }
    return true;
}

int main()
{
    cout << isPrime(9);
    return 0;
}
`

**Comment by mrKumarAtul** [comment]
Yes i do agree with this point, i also witnessed the same issue and was about to open a new issue, thanks for mentioning it here.

**Comment by josehidalgoromero** [comment]
You forget to apply the right operator in the loop:

`#include <iostream>
#include <cmath> /// We need the "cmath" library for the sqrt function

using namespace std;

bool isPrime(int n) {
    if(n <= 1)
        return false;
    for(int i = 2; i <= sqrt(n); i++) {
        if(n % i == 0)
            return false;
    }
    return true;
}

int main()
{
    cout << isPrime(9);
    return 0;
}`

**Comment by FahimFBA** [comment]
I fixed that in Pull Request #5
